src/Nuget.PackageIndex/ISettingsJson.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Collections.Generic;
+
+namespace Nuget.PackageIndex
+{
+    /// <summary>
+    /// Index settings
+    /// </summary>
+    public interface ISettingsJson
+    {
+        List<string> IncludePackagePatterns { get; }
```

Comment:
Don't put List<T> on interfaces directly. If it is meant to be changed use IList, if its readonly use IReadOnlyList